src/scripts/common.js

```diff
@@ -512,66 +502,58 @@ window.togglbutton = {
     // Add created link to links array
     togglbutton.links.push({ params: params, link: link });
 
-    // new button created - set state
+    return link;
+  },
+
+  // Query active timer entry, and set it to active. If a callback is
+  // provided as an argument, it will be invoked before updating the
```

Comment:
It seems that a callback isn't an argument here in `queryAndUpdateTimerLink`, I assume it wasn't needed after all?